worker/runtime/rootfs_manager_test.go

```diff
@@ -81,9 +72,90 @@ func (s *RootfsManagerSuite) TestSetupCwdWithoutIDMappings() {
 		}),
 	)
 
-	err := mgr.SetupCwd(s.baseSpec, "dir")
+	err := mgr.SetupCwd(s.rootfsPath, "dir")
 	s.NoError(err)
 
 	s.Equal(expectedPath, path)
 	s.Equal(expectedMode, mode)
 }
+
+func (s *RootfsManagerSuite) TestLookupUserReturnsUserInfo() {
+	mgr := runtime.NewRootfsManager()
+	s.writeEtcPasswd(`
+		root:*:0:0:System Administrator:/var/root:/bin/sh
+		some_user_name:*:1:1:Some User:/var/root:/usr/bin/false
+	`)
+	actualUser, ok, err := mgr.LookupUser(s.rootfsPath, "some_user_name")
+	s.NoError(err)
+
+	s.True(ok)
+	expectedUser := specs.User{
+		UID: 1,
+		GID: 1,
+	}
+	s.Equal(expectedUser, actualUser)
+}
+
+func (s *RootfsManagerSuite) TestLookupUserUsernameNotFound() {
+	mgr := runtime.NewRootfsManager()
+
+	s.writeEtcPasswd(`
+		root:*:0:0:System Administrator:/var/root:/bin/sh
+		some_user_name:*:1:1:Some User:/var/root:/usr/bin/false
+	`)
+
+	_, ok, err := mgr.LookupUser(s.rootfsPath, "missing_username")
+	s.NoError(err)
+	s.False(ok)
+}
+
+func (s *RootfsManagerSuite) TestLookupUserInvalidUID() {
+	mgr := runtime.NewRootfsManager()
+
+	s.writeEtcPasswd(`
+		some_user_name:*:NaN:0:System Administrator:/var/root:/bin/sh
+	`)
+
+	_, _, err := mgr.LookupUser(s.rootfsPath, "some_user_name")
+	s.Error(err)
```

Comment:
It would be clearer to assert not only an error was returned but it matched a type or error message "invalid uid....."
Same for the other two cases below

Reply:
Good point! Done.